package.json

```diff
@@ -36,7 +36,7 @@
     "@abi-software/flatmapvuer": "^0.5.4",
     "@abi-software/map-side-bar": "^1.4.1-beta.1",
     "@abi-software/plotvuer": "^0.3.9",
-    "@abi-software/scaffoldvuer": "^0.2.1",
+    "@abi-software/scaffoldvuer": "^0.2.2",
```

Comment:
was the scaffoldvuer change related to this?

Reply:
There are some changes related to a Portal ticket in it.